### ISSUE TITLE
Calculating OKS for custom dataset.

### ISSUE THREAD

**Issue by TobiasJohansen** [cocodataset/cocoapi#278]
Hi, I'm working with a custom dataset, using COCO annotation, containing labeled data of lobsters and want to tune the per keypoint value k_i  to fit the keypoints I have. The images I have are shot from above, resulting in images of lobsters facing all directions. I have tried writing code to center, scale and rotate the annotations I have and calculate the standard-deviations from this but I'm not sure if this is the right way to do it?
My question is how are the standard-deviations calculated for humans? Im having trouble understanding the specifics.

**Comment by yadunathg** [comment]
I have a related question. [OKs standard deviation values](https://github.com/cocodataset/cocoapi/blob/master/PythonAPI/pycocotools/cocoeval.py#L523) are hard coded with no comments on how they are calculated. How do I calculate these values for my custom dataset?

**Comment by harrisonford** [comment]
Yes they hardcoded them and they're pretty much empirically found, their explanation is found [here](http://cocodataset.org/#keypoints-eval) on "1.3 Tuning OKS".

So in your case I guess you should take some pictures and measure some standard deviations yourself and check what's the best values, this method is very unsophisticated IMO.

**Comment by yadunathg** [comment]
Thanks for the reference

**Comment by tucachmo2202** [comment]
> Yes they hardcoded them and they're pretty much empirically found, their explanation is found [here](http://cocodataset.org/#keypoints-eval) on "1.3 Tuning OKS".
> 
> So in your case I guess you should take some pictures and measure some standard deviations yourself and check what's the best values, this method is very unsophisticated IMO.

Hi, I'm not clear with this reference. I don't understand this line "First, using 5000 redundantly annotated images in val, for each keypoint type i we measured the per-keypoint standard deviation σi with respect to object scale s.". What can I do to measured the per-keypoint standard deviation σi?
